test/testutil/testutil.go

```diff
@@ -105,3 +107,32 @@ func ParseJsonStrict[T any](input io.Reader) (T, error) {
 	decoder.DisallowUnknownFields()
 	return parsedStdout, decoder.Decode(&parsedStdout)
 }
+
+// CaptureConsoleOutput borrows from `github.com/zenizh/go-capturer`. This implementation captures both stdout
+// and stderr. Consider adding the go-capturer module if more granularity is needed
+func CaptureConsoleOutput(f func()) string {
```

Comment:
I think there's already something very similar to this in the integration test util file; I used it when we launch the CLI as an external process and want to capture stdout/stderr